js/screens/welcome-screen.js

```diff
@@ -1,5 +1,54 @@
+import { wizardScreen } from './wizard-screen.js';
+/**
+ * This function creates the welcome screen..
+ *
+ */
 export const welcomeScreen = () => {
+   // 1. Establish main wrapper.
    const welcomeStepWrapper = document.createElement('div');
-   welcomeStepWrapper.innerText = 'Welcome screen';
+   welcomeStepWrapper.classList.add('welcomeScreen');
+
+   // 2. Create all the required elements.
+   let gameNameText = document.createElement('span');
+   let nameFirstPart = document.createElement('span');
+   let nameSecondPart = document.createElement('span');
+   let swordsImg = document.createElement('img');
+   let infoText = document.createElement('span');
+   let button = document.createElement('img');
```

Comment:
**issue:** Button should not be an `img` tag. Using the <button> element instead of an <img> tag for buttons is recommended due to better accessibility support, proper semantic representation, and built-in event handling capabilities